Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "ads1x1x"
-version = "0.2.2" # remember to update html_root_url
+version = "0.3.0"
```

```suggestion
version = "0.2.2"
```
Let's do version bumping separately.